test/parallel/test-debug-port-from-cmdline.js

```diff
@@ -38,11 +38,12 @@ function processStderrLine(line) {
 
 function assertOutputLines() {
   const expectedLines = [
-    'Starting debugger agent.',
-    'Debugger listening on 127.0.0.1:' + debugPort,
+    /Starting debugger agent./,
+    /\(node:\d+\) DeprecationWarning:.*/,
```

Comment:
Might be better to use `common.expectWarning()` (I may have the name wrong... there's a function for this tho :-) ..)